### ISSUE TITLE
BF4 gets no route available

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/raiden#5375]
## Problem Definition

Scenario in a nutshell:

node0: `0xf8Af92870eF35F00093609CedB7bae463c3FF819`
node1: `0x34F4A17C8743d89Ed1982803b1a1014761a28cA2`
node2: `0xfD3dc60ec73D48318FcC7A7a08243BD76917104b`
node3: `0x76eA09e1fE1a16770bD2AD797019C066C1635791`
node4: `0xDc4B6C710F1D1Ddf2977cE0924B35F6A333D34b2`
node5: `0xF6255Dff6EfD5Be3f17F3262721E60397a683b54`

```
{
  "task": "<TransferTask: {'from': 0, 'to': 5, 'amount': 1000000000000000, 'lock_timeout': 30}>",
  "event": "Task errored",
  "logger": "scenario_player.tasks.base",
  "level": "error",
  "timestamp": "2019-11-26 08:18:18.043895",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 74, in __call__\n    return self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/api_base.py\", line 59, in _run\n    f'HTTP status code \"{resp.status_code}\" while fetching {url}. '\nscenario_player.exceptions.legacy.RESTAPIStatusMismatchError: HTTP status code \"409\" while fetching http://127.0.0.1:32919/api/v1/payments/0x62083c80353Df771426D209eF578619EE68D5C7A/0xF6255Dff6EfD5Be3f17F3262721E60397a683b54. Expected 2..: {\"errors\": \"Payment couldn't be completed because: there is no route available\"}"
}
```
what does the exception say:

1. `Payment couldn't be completed because: there is no route available\"`

2. Occurs while requesting

`http://127.0.0.1:32919/api/v1/payments/0x62083c80353Df771426D209eF578619EE68D5C7A/0xF6255Dff6EfD5Be3f17F3262721E60397a683b54`
where

`0x62083c80353Df771426D209eF578619EE68D5C7A`: token address
`0xF6255Dff6EfD5Be3f17F3262721E60397a683b54`: node5 address

3. Scenario fails at
`- transfer: {from: 0, to: 5, amount: 1_000_000_000_000_000, lock_timeout: 30}`
line 86

https://drive.google.com/file/d/1tPkOXiT2DHu5d8nGP94G0UDVhfIrlgZt/view?usp=sharing

**Comment by Dominik1999** [comment]
Might be a duplicate to #5337

**Comment by Dominik1999** [comment]
Closed and merged into META Issue #5433
